### ISSUE TITLE
[Native][Aggregate functions] Implement set_agg(x)

### ISSUE THREAD

**Issue by yzhang1991** [prestodb/presto#18855]
Ahana issue link: https://ahana-io.atlassian.net/browse/PDB-352

**Comment by czentgr** [comment]
Implemented in velox with PR https://github.com/facebookincubator/velox/pull/5191 for numeric types tinyint, smallint, integer, bigint.

**Comment by czentgr** [comment]
Right, I just wanted to point out that some work was already done. As the Presto doc doesn't indicate which types are supported by the function - the assumption is all types must be supported to close the issue in which case more work is needed.

**Comment by mbasmanova** [comment]
@czentgr I added support for all types: https://github.com/facebookincubator/velox/pull/5445
